src/routes/auth.js

```diff
@@ -0,0 +1,66 @@
+import { Router } from 'express';
+
+import { createUser, login } from './controllers/auth';
+
+const authRouter = Router();
+
+/**
+ * @swagger
+ *
+ * /api/v1/users:
+ *   post:
+ *     description: User Registration Endpoint
+ *     produces:
+ *       - application/json
+ *     request:
+ *         content:
+ *         - application/json
+ *         schema:
+ *           type: array
+ *           items:
+ *         $ref: '#/definitions/users'
+ *     responses:
+ *       201:
+ *         description: User created
+ *       400:
+ *         description: Bad request.
+ *       401:
+ *         description: Authorization information is missing or invalid.
+ *       404:
+ *        description: A user with the specified ID was not found.
+ *       5XX:
+ *        description: Unexpected error.
+ */
+authRouter.post('/', createUser);
```

Comment:
please can rename the name of this controller to `signup` and route to `/signup`

Reply:
Please know that both signup and login are in the same controller.

For clarity, are you suggesting that auth.js be changed to signup.js @Japhethca ?

Reply:
No, I meant the route and controller name.

Reply:
Okay.